src/token/token_parser.h

```diff
@@ -4,3 +4,15 @@
 #include "../common/buffer.h"
 
 bool parse_token(buffer_t *buf, token_t *token);
+
+/**
+ * Find the index of a token on the token registry
+ *
+ * The token registry is filled with calls to SEND_TOKEN_DATA
+ *
+ * @param[in] uid
+ *   The uid of the token we want to find
+ *
+ * @return index of the token on registry or -1 if not on registry
```

Comment:
`int8_t` is enough because `TX_MAX_TOKENS = 10 < 128`, right? I think this would be useful to mention here.

Reply:
Yes, and I use `int8_t` instead of `uint8_t` because we return `-1` when not found.
I'll add this explanation.

Reply:
Thanks!